tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -61,6 +71,16 @@ Compose kill
     Log  ${out}
     Should Be Equal As Integers  ${rc}  0
 
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network inspect vic_default
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+    Wait Until Keyword Succeeds  1 min  5 sec  Check Container Removed From Network  vic_redis_1  vic_default
+
```

Comment:
I don't necessarily like this... what are we asking the user to do here? I would think that in our response to users is to run the compose down again? So why don't we just run compose down until it succeeds for a reasonable number of times?

Reply:
No, I don't think we can run compose down here. 

If you run `docker-compose down`, it will first remove the container and then remove the network. If the vsphere event for the container being powered off is received after `compose down` removes the container, we may fail to unbind it from the network (container does not exist). And the network will always have this dangling container (although it is already removed from cache).

Reply:
So at that point, after doing a perfectly normal set of steps, the customer is screwed and cannot do anything?